.github/ISSUE_TEMPLATE/lyrics-error.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Lyrics error
+about: Report a mistake in the lyrics we have.
+title: "[LYRICS] "
+labels: lyrics
```

Comment:
```suggestion
labels: lyrics mistake
```

Reply:
???? there is only the [lyrics] tag/label. you made it

Reply:
I added the label before suggesting

Reply:
![image](https://user-images.githubusercontent.com/44261356/177013118-13c298ec-a972-4f31-8632-bb5d65547c32.png)